js/menu.js

```diff
@@ -235,6 +235,29 @@ function closed() {
   popWin.style.visibility = 'hidden';
 }
 
+// Add_validation_form
+function validateEmail(event) {
+  event.preventDefault();
+  const email = document.forms[0].elements.Email.value;
+  // check email
+  const res = email.match(/[A-Z]/g);
+  const btn = document.querySelector('form #form_btn');
+  const container = document.querySelector('form #form-btn-container');
+  const child = document.createElement('small');
+  let msg = '';
+
+  if (res) msg = 'Error:Form was not Sent! The email should be lowercase only';
```

Comment:
- [ ] Hi @Kanyorok, great job for validating the email field. When I enter the correct email, no error message is displayed and no success message is displayed. The form is not sent. Kindly make sure that when click on submit after filling out the fields correctly, the form should be submitted.